specification/templates/.github/CODEOWNERS

```
@@ -4,3 +4,14 @@
 * @signalfx/gdi-TEAM-maintainers @signalfx/gdi-TEAM-approvers
 
 .github/CODEOWNERS @signalfx/gdi-TEAM-maintainers
+
+#####################################################
+#
+# Docs reviewers
+#
+#####################################################
+
+*.md @signalfx/docs
+*.rst @signalfx/docs
+docs/ @signalfx/docs
+README @signalfx/docs @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
```

I think most often it is `README.md`.

```suggestion
README* @signalfx/docs @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
```